dpnp/dpnp_iface_linearalgebra.py

```diff
@@ -166,15 +167,59 @@ def dot(a, b, out=None):
     return dpnp.get_result_array(result, out, casting="no")
 
 
-def einsum(*args, **kwargs):
+def einsum(*operands, out=None, optimize=False, **kwargs):
```

Comment:
Should not we use the exact keyword rather then `kwargs` here?
```suggestion
def einsum(*operands, out=None, dtype=None, order="K", casting="safe", optimize=False):
```